src/util/message.cpp

```diff
@@ -0,0 +1,63 @@
+#include <faabric/util/message.h>
+
+namespace faabric::util {
+void copyMessage(const faabric::Message* src, faabric::Message* dst)
+{
+    dst->set_id(src->id());
```

Comment:
This function will rot within the next week or so, see my comment above re. using assignment.

Reply:
The copy constructor of protobuf objects also does a deep copy already, it's even used in a couple of places in this codebase